README.md

```diff
@@ -172,27 +188,79 @@ command.
 There are three views for reviewing a pull request on GitHub: **Conversation**,
 **Commits**, and **Files changed**.
 
-#### Conversation ####
+**Conversation**
+
+- Comments on the pull request show up here, including any inline comments that
+  are added when viewing the file diffs. A summary of submitted commits is also
+  listed.
+- Use this section to submit general feedback about the pull request.
+
+**Commits**
+
+- A more detailed list of commits included in the pull request, that will allow
+  a reviewer to view each individually.
+- Usually before merging, you'll want all the commits from that change to be
+  squashed into a single commit. This so that the commit can show the whole
+  change that was made, and so that there aren't a bunch of typo fixes in the
+  repository's commit history!
+
+**Files changed**
 
-Comments on the pull request show up here, including any inline comments that
-are added when viewing the file diffs. A summary of submitted commits is also
-listed.
+- The `diff` view of the pull request. Lets the reviewer see the full patch in
+  its entirety, and add inline comments on changes.
+- Use this section to submit line-specific feedback.
 
-Use this section to submit general feedback about the pull request.
+#### Giving feedback ####
 
-#### Commits ####
+Everyone is invited to give feedback regardless of experience. Comments can
+touch on:
 
-A more detailed list of commits included in the pull request, that will allow a
-reviewer to view each individually.
+- phrasing of any text added
+- tags on event posts (do they make sense? are they existing tags?)
+- lines that are over 80 characters long (which we avoid)
+- any identation that's hard to read 
+- the way the change looks on the [Heroku Review App](#heroku-review-apps)
+- if there are mutliple commmits that can be [squashed into one
+  commit](#squashing-commits)
 
-#### Files changed ####
+You can even look at it and decide it looks good and just comment with “looks
+good to me!” 
 
-The `diff` view of the pull request. Lets the reviewer see the full patch in
-its entirety, and add inline comments on changes.
 
-Use this section to submit line-specific feedback.
+### Squashing commits ###
+
+If you have `x` commits you want to squash into one, run `[git
+rebase](https://help.github.com/articles/about-git-rebase/) -i HEAD~x`. It will
+list the last `x` commits. Keep the first one as `pick`, and then choose
+`fixup` (or you can just type `f`) for the rest of the commits. Save and close,
+and there should just be one commit there! (To confirm, you can run `git log`.)
+
+When you push the new squashed commit, `git` will be confused and think you're
+missing those `x` commits (when really you just turned them all into a new
+single commit). You might see the error message failed to push some refs to
+`https://github.com/<some-path>` and a suggestion to run `git pull`. **Do not
+ever run `git pull`**. Instead run `git push --force`.
```

Comment:
"Do not ever run git pull. It will bring back in a copy of all the commits you squashed and potentially other commits unrelated to your change. Instead..."

how's that?